browser/js/user-dataset-details/user-dataset-details.html

```diff
@@ -0,0 +1,10 @@
+<div class="dataset-container">
+	<header class="text-center">
+		<h2>{{ dataset.name || dataset.resource.name }}</h2>
```

Comment:
Is there any way to standardize how the dataset comes in here to make the OR unnecessary?